### PR TITLE
Fix develop branch merge error

### LINKED ISSUE #290: Fix develop branch merge error
**Branch:** `develop (os x installers)

- [x] use /Library/LaunchAgents/ as .plist destination
- [x] use /usr/local/bin/ as kalite executable destination

### PR DESCRIPTION
**Pull request by amodia**
@cpauya this fixes #290

### CODE REVIEW

File: osx/post_installation.sh [hunk 1]
```diff
@@ -34,14 +34,14 @@ ASSESSMENT_SRC="$KALITE_SHARED/assessment/assessment.zip"
...
 COMMAND_SYMLINK="ln -sf $SYMLINK_FILE $SYMLINK_TO"
 
 ORG="org.learningequality.kalite"
-LAUNCH_AGENTS="$HOME/Library/LaunchAgents/"
+LAUNCH_AGENTS="/Library/LaunchAgents/"
 KALITE=$(which kalite)
-PLIST_SRC="$LAUNCH_AGENTS$ORG.plist"
-
+TMP="/tmp/"
+PLIST_SRC="$TMP$ORG.plist"
```
Comment (cpauya) [inline osx/post_installation.sh]: We already removed the use of temporary directory.  Pls remove this.

File: osx/post_installation.sh [hunk 2]
```diff
@@ -124,7 +124,19 @@ if [ ! -d "$LAUNCH_AGENTS" ]; then
...
+if [ ! -d "$LAUNCH_AGENTS" ]; then
+    echo ".. Must create '$LAUNCH_AGENTS' folder..."
+    sudo mkdir -p $LAUNCH_AGENTS
+    if [ $? -ne 0 ]; then
+        echo ".. Abort!  Error encountered creating '$LAUNCH_AGENTS' directory."
+        exit 1
+    fi
+fi
 create_plist
+sudo cp $PLIST_SRC $LAUNCH_AGENTS
```
Comment (cpauya) [inline osx/post_installation.sh]: This one too.  If you want to test, just run with `sudo ./post_installation.sh` @amodia.

File: osx/post_installation.sh [hunk 3]
```diff
@@ -124,7 +124,19 @@ if [ ! -d "$LAUNCH_AGENTS" ]; then
     fi
 fi
 
+
+((STEP++))
+echo "$STEP/$STEPS. Create plist in /Library/LaunchAgents folders..."
+if [ ! -d "$LAUNCH_AGENTS" ]; then
```
Comment (cpauya) [inline osx/post_installation.sh]: We need this check if we are using `~/Library/LaunchAgents/` but I think there's no need for this if we are using `/Library/LaunchAgents/`.

**Comment by amodia** [comment]
@cpauya it's working, tested on OS X EL Capitan.

screen shot 2015-12-02 at 8 25 46 pm

File: osx/post_installation.sh (hunk 3 above)
Comment (cpauya) [inline osx/post_installation.sh]: Ok, wouldn't hurt to double-check if it's there. :smile_cat:

**Comment by cpauya** [comment]
Merging.

**Pull request merged**